taxi/models.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.conf import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    country = models.CharField(max_length=100)
+
+
+class Driver(AbstractUser):
```

Comment:
According to the checklist, you should use `settings.AUTH_USER_MODEL` instead of `AbstractUser` for user model reference. This is a best practice in Django to avoid hardcoding the user model and allows flexibility in case the user model changes in the future.